terraform/environments/mlra/modules/alb/main.tf

```diff
@@ -122,11 +146,11 @@ data "aws_elb_service_account" "default" {}
 resource "aws_lb" "loadbalancer" {
   #checkov:skip=CKV_AWS_150:preventing destroy can be controlled outside of the module
   #checkov:skip=CKV2_AWS_28:WAF is configured outside of the module for more flexibility
-  name                       = "${var.application_name}-application-lb"
+  name                       = "${var.application_name}-application-external-lb"
   internal                   = var.internal_lb
   load_balancer_type         = "application"
   security_groups            = [aws_security_group.lb.id]
-  subnets                    = [var.private_subnets[0], var.private_subnets[1], var.private_subnets[2]]
+  subnets                    = [var.public_subnets[0], var.public_subnets[1], var.public_subnets[2]]
   enable_deletion_protection = var.enable_deletion_protection
```

Comment:
Work done to make ALB external, for https://dsdmoj.atlassian.net/browse/LAWS-3073